File: web/src/app/models/article.ts
import { Translation } from './translation';

export class Article {
    id: string;
    date: Date;
    author: string[];
    category: string;
    topics: string[];
    title: string;
    subtitle: string;
    summary: string;
    language: string;
    sourceLink: string;
    words: number;
    characters: number;
    owner: string;
    hasText: boolean;
    textLink: string;
    imageLink: string;
    translations: Translation[];
}

File: api/DesiringGodArticlesCrawler/Models/Article.cs
using System;
using System.Collections.Generic;

namespace DesiringGodArticlesCrawler.Models
{
    public class Article
    {
        public List<string> Author { get; set; }
        public string Category { get; set; }
        public string Title { get; set; }
        public string Subtitle { get; set; }
        public string Link { get; set; }
        public string ImageLink { get; set; }
        public DateTime Date { get; set; }
        public string Summary { get; set; }
        public string Scripture { get; set; }
        public List<string> Topics { get; set; }
        public string Text { get; set; }
    }

    public class ArticleOld
    {
        public string Author { get; set; }
        public string Category { get; set; }
        public string Title { get; set; }
        public string Subtitle { get; set; }
        public string Link { get; set; }
        public string ImageLink { get; set; }
        public DateTime Date { get; set; }
        public string Summary { get; set; }
        public string Scripture { get; set; }
        public List<string> Topics { get; set; }
        public string Text { get; set; }
    }
}
